### PR TITLE
[Merged by Bors] - chore: split Topology.Algebra.Module.WeakDual

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Module/WeakDual.lean [hunk 1]
```diff
@@ -3,28 +3,29 @@ Copyright (c) 2021 Kalle Kytölä. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kalle Kytölä, Moritz Doll
 -/
-import Mathlib.Topology.Algebra.Module.Basic
+
```
Comment (YaelDillies) [inline Mathlib/Topology/Algebra/Module/WeakDual.lean:6]:
```suggestion
```

**Review by YaelDillies — COMMENTED** [review]
@kkytola, are you happy with this split?

**Comment by YaelDillies** [comment]
In the lack of dissonant voices, I hereby declare this split is fine. See [Zulip](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.2316489.20.20chore.3A.20split.20Topology.2EAlgebra.2EModule.2EWeakDual).

maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**